packages/react/src/search/search.react.tsx

```diff
@@ -0,0 +1,22 @@
+import { c, classy } from '@onfido/castor';
+import { Icon } from '@onfido/castor-react';
+import React from 'react';
+import { Input, InputProps } from '../input/input.react';
+import { withRef } from '../utils';
+
+/**
+ * @note `Search` uses an `Icon` that requires
+ * `Icons` (SVG sprite) to be included in your app.
+ *
+ * https://github.com/onfido/castor-icons#use-with-plain-code
+ */
+export const Search = withRef(
+  ({ className, style, ...props }: SearchProps, ref: SearchProps['ref']) => (
+    <div className={classy(c('search'), className)} style={style}>
+      <Input {...props} ref={ref} type="search" />
```

Comment:
To keep consistency across component code style:

```suggestion
  ({ className, style, ...restProps }: SearchProps, ref: SearchProps['ref']) => (
    <div className={classy(c('search'), className)} style={style}>
      <Input {...restProps} ref={ref} type="search" />
```

Reply:
Do you prefer `restProps` or would you mind if I created another PR to rename them all to `props`?

Reply:
I prefer `restProps`. Reason for this that `props` come as an argument which here is destructed, and what's left is `restProps`. So `className + style + restProps = props`.